app/src/main/java/com/eveningoutpost/dexdrip/AlertList.java

```diff
@@ -159,9 +173,36 @@ public void onClick(View v) {
         });
     }
 
+    void displayWarning() {
+        runOnUiThread(new Runnable() {
+            @Override
+            public void run() {
+
+                if (!isFinishing()){
+                    new AlertDialog.Builder(AlertList.this)
+                      .setTitle("Warnining !")
+                      .setMessage("No active low alert exists, it is highly recomended that you will add a low alert.")
```

Comment:
Can we change the message to: "No active Low Alert exists, without this there will be no alert on low glucose! Please add or enable a low alert."